bin/writeNextflowConfig

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/perl
```

Comment:
why is this file in both the biom and isasimple plugin?  is that code duplication?